mobile/android/app/build.gradle

```diff
@@ -57,21 +57,18 @@ android {
         versionName flutterVersionName
     }
 
-  //  signingConfigs {
-  //      release {
-  //          keyAlias keystoreProperties['keyAlias']
-  //          keyPassword keystoreProperties['keyPassword']
-  //          storeFile keystoreProperties['storeFile'] ? file(keystoreProperties['storeFile']) : null
-  //          storePassword keystoreProperties['storePassword']
-  //      }
-  //  }
+   signingConfigs {
+       release {
+           keyAlias "$System.env.ALIAS" ? keystoreProperties['keyAlias'] : null
+           keyPassword "$System.env.ANDROID_KEY_PASSWORD" ? keystoreProperties['keyPassword'] : null
+           storeFile file("../key.jks") ? file(keystoreProperties['storeFile']) : null
+           storePassword "$System.env.ANDROID_STORE_PASSWORD" ? keystoreProperties['storePassword'] : null
```

Comment:
Am I understanding these right? If ALIAS is defined use `keystoreProperties` otherwise use null? Wouldn't you want to use the env value if it's defined, otherwise use keystore properties?

Reply:
Yeah, brainfart